distribution/lib/Standard/Table/0.0.0-dev/src/Io/Excel.enso

```diff
@@ -66,10 +66,18 @@ type Excel_Range
     to_text = "Excel_Range " + this.address
 
     ## Validates if a column index (1-based) is within the valid range for Excel.
+
+       Arguments:
+       - column: 1-based index to check
+       - limit: maximum valid index, defaults to Excel 2007+ limit of 16,384
```

Comment:
I still don't understand why `limit` is not a constant. You never seem to use an alternative value for it. What scenarios do you imagine where we would re-use this function and override this second argument?

Reply:
Excel 95 format for example is 256 and 65,536 so I imagined switching between the two but hadn't enforced this yet

Reply:
Ah, I see.

Still, it feels a bit weird to me to have a constant saved as a default parameter.

If you think it should be parametrized, I'd suggest:
```
type Excel_Version
     type Excel_95
     type Excel_2003 # or sth

     maximum_column_index : Integer
     maximum_column_index = case this of
         Excel_95 -> 256
         Excel_2003 -> 16384

...

is_valid_column : Integer -> Excel_Version -> Boolean
is_valid_column x version=Excel_2003 = ...
```

Reply:
Will remove the constant then